### ISSUE TITLE
FPS Issue

### ISSUE THREAD

**Issue by ghost** [belathus/eldritch_revolution#5]
I don't know if this is just my computer or the pack itself, but I felt I should let you know just in case. A problem that I have is that when in a world or even the title-screen, my frames per second are anywhere between 0 and 10. However, when in a menu, like the settings, for example, my fps is fine. I tried installing Optifine, but I couldn't get it to run with it. 
I really hope that either this is fixed, or someone might be able to help me, as I really want to play this pack.
Thank you!

**Comment by Divineaspect** [comment]
Not sure what may be up with optifine.
Do you get this issue with more then one single player world?

**Comment by ghost** [comment]
Yeah, every world is like this. As soon as I hit escape, however, my mouse moves around fine, and I can interact with the buttons without a problem. The inventory is choppy as well. For some reason, it's also like that when I open the menus straight from the main menu.

**Comment by Divineaspect** [comment]
how much memory do you have assigned?
Have you added any other mods?

**Comment by ghost** [comment]
I have the base mods (obviously), and from the optional ones, I have Walia, Walia Harvestability, JourneyMap,and Item Scroller. Other than those options, I have added no mods on my own.

**Comment by ghost** [comment]
I don't know what fixed it, but I've got it to work at 30-ish fps. I added BetterFPS and reduced the Mipmap levels to OFF. Hope that might shine some light on the situation
